applications/pelion_client/README.rst

```diff
@@ -154,7 +154,7 @@ The following table lists modules that are part of the application.
 
 All modules react to ``module_state_event``.
 
-On top of these modules, the application uses the following modules from :ref:`lib_caf` (CAF), a set of generic modules based on the Event Manager and available to all applications in the |NCS|:
+On top of these modules, the application uses the following modules :ref:`lib_caf` (CAF), a set of generic modules based on the Event Manager and available to all applications in the |NCS|:
```

Comment:
Why remove `from`? Common Application Framework is the name of the set of modules.

Reply:
Oops - thanks for catching this!